src/alert.html

```diff
@@ -1,14 +1,13 @@
-@@include('header.htm', { "title": "Sleek - Admin Dashboard Template", "className": "header-fixed sidebar-fixed sidebar-dark header-light" })
+@@include('header.htm', { "title": "Sleek - Admin Dashboard Template", "className": "header-fixed sidebar-fixed sidebar-dark header-light right-sidebar-content-close" })
 @@include('sidebar-with-footer.htm', { "parent": "ui-elements", "sub_parent": "components", "active": "alert" })
 
 		<!-- ====================================
 		——— LAYOUT CONTAINER
```

Comment:
Update name in comment so user can understand better

Reply:
Ok i have fixed